src/qt/overviewpage.cpp

```diff
@@ -211,7 +211,10 @@ void OverviewPage::setPrivacy(bool privacy)
         coinJoinStatus(true);
     }
 
+    // We can't hide the frame itself as it would disturb alignment, hide inner elements instead
     ui->listTransactions->setVisible(!m_privacy);
+    ui->label_4->setVisible(!m_privacy);
+    ui->labelTransactionsStatus->setVisible(!m_privacy);
```

Comment:
https://github.com/dashpay/dash/pull/6829/commits/4fe0be8ebc9e25a22a4e523786f27197a1f43c72#r2310778220: good catch by @coderabbitai but I think it should be more like
```suggestion
    ui->labelTransactionsStatus->setVisible(!m_privacy && ui->labelWalletStatus->isVisible());
```